app/containers/Bookings/Details/QuickLook/index.js

```diff
@@ -206,52 +206,62 @@ NegativeAndPositiveCaseNoteCount.defaultProps = {
   positiveCaseNotes: '',
 }
 
-export const Adjudications = ({ adjudications }) => {
+export const Adjudications = ({ adjudications, adjudicationHistoryUrl }) => {
   const awards = adjudications.get('awards')
   const proven = adjudications.get('proven')
   return (
-    <div>
+    <>
       <div className="row border-bottom-line">
-        <div className="col-lg-6 col-xs-6">
-          <span>Proven adjudications</span>
-        </div>
-
-        <div className="col-lg-6 col-xs-6">
-          <b> {proven || 0} </b>
-        </div>
-      </div>
-
-      {awards.size === 0 && (
-        <div className="add-gutter-margin-top">
+        <div className="row border-bottom-line">
           <div className="col-lg-6 col-xs-6">
-            <span>Active adjudications</span>
+            <span>Proven adjudications</span>
           </div>
 
           <div className="col-lg-6 col-xs-6">
-            <b> No active awards </b>
+            <b> {proven || 0} </b>
           </div>
         </div>
-      )}
-
-      {awards.map((award, index) => (
-        <div key={uuid()} className="row add-gutter-margin-top">
-          <div className="col-lg-6 col-xs-6">{index === 0 && <span>Active awards</span>}</div>
-          <div className="col-lg-6 col-xs-6">
-            <b>
-              {' '}
-              {award.get('durationText')} {award.get('sanctionCodeDescription')}{' '}
-            </b>
-            <div> {award.get('comment')} </div>
-            <div> {award.get('effectiveDate') && <FormattedDate value={award.get('effectiveDate')} />} </div>
+        {awards.size === 0 && (
+          <div className="add-gutter-margin-top">
+            <div className="col-lg-6 col-xs-6">
+              <span>Active adjudications</span>
+            </div>
+            <div className="col-lg-6 col-xs-6">
+              <b> No active awards </b>
+            </div>
           </div>
-        </div>
-      ))}
-    </div>
+        )}
+        {awards.map((award, index) => (
+          <div key={uuid()} className="row add-gutter-margin-top">
+            <div className="col-lg-6 col-xs-6">{index === 0 && <span>Active awards</span>}</div>
+            <div className="col-lg-6 col-xs-6">
+              <b>
+                {' '}
```

Comment:
Could probably do away with the spaces `{' '}`

Reply:
removed these